android/assets/jsons/translations/template.properties

```diff
@@ -1463,6 +1463,12 @@ Spy Hideout =
 Location = 
 Spy present = 
 Move = 
+None = 
+Moving = 
```

Comment:
Please add these dynamically in the translationfilewriter by iterating on the spy action enum
There are a couple of other enums we do this with

Reply:
Should now be fixed :)